docs/Compile.rst

```diff
@@ -338,6 +338,33 @@ from your GCC version and distribute that too, it will fail on older OS X
 versions.) For this reason, if you plan on distributing DFHack, it is highly
 recommended to use GCC 4.8 or 7.
 
+Notes for M1 users
+------------------
+
+Alongside the above, you will need to follow these additional steps to get it
+running on Apple silicon.
+
+Install an x86 copy of ``homebrew`` alongside your existing one. `This
+stackoverflow answer <https://stackoverflow.com/a/64951025>` describes the
```

Comment:
I believe you need a `_` or `__` after the link here. I usually go with the latter because it avoids issues with using the same link more than once.

```suggestion
stackoverflow answer <https://stackoverflow.com/a/64951025>`__ describes the
```

(Since it seems like you might be familiar with RST: we've [configured](https://github.com/DFHack/dfhack/blob/0aeb10784431db91c0e0186829fd627c2134f1e2/conf.py#L281) our default role as `ref`, so single backticks without an extra prefix or suffix always point to named anchors internal to our docs)